subsys/bluetooth/services/ots/ots_client.c

```diff
@@ -1,13 +1,11 @@
 /** @file
  *  @brief Bluetooth Object Transfer Client
  *
- * Copyright (c) 2020 - 2021 Nordic Semiconductor ASA
+ * Copyright (c) 2020-2022 Nordic Semiconductor ASA
```

Comment:
This single implementation file could be divided into smaller implementation units (similar to the OTS server implementation). Potential candidates:

- Module for object navigation ots_olcp_client.c
- Module for object manipulation ots_oacp_client.c

Reply:
That would make for an implementation more similar to the OTS, and could be a good idea.
Propose to postpone for now: https://github.com/zephyrproject-rtos/zephyr/issues/42820